src/index.ts

```diff
@@ -6396,6 +6398,17 @@ const shardusSetup = (): void => {
           shardus.shutdownFromDapp(tag, message, false)
           return false
         }
+
+        //error out nodes in debug mode for production networks to prevent joining
+        if (
+          networkAccount.data.mode === ServerMode.Release &&
```

Comment:
in the above else condition to fetch the network account from the archiver, will this network account also have a `mode` field?

Reply:
so was the mode field already part of the networkAccount we get from the archiver?

Reply:
I meant, once I added it to the Network account obj here (https://github.com/shardeum/shardeum/pull/42/files#diff-a9f7b5615265a25a5c043bea768b8d1fa86baf642022b1b7324dba2708145d7cR19) it was stored, picked up by the archiver and appeared on the archiver response as expected

Reply:
ok makes sense